tests/test_create_save_summary.py

```diff
@@ -242,8 +244,13 @@ def test_generate_coding_summary(reads, data_folder, single_alphabet_ksize_true_
         "peptide_ksize": 7,
         "jaccard_threshold": 0.5,
     }
-
-    assert test_summary == true_summary
+    for key, value in test_summary.items():
+        if type(value) is str:
+            assert value == true_summary[key]
+        elif type(value) is float:
+            assert_almost_equal(value, true_summary[key], decimal=15)
```

Comment:
ah - was this the actual issue with the 75% value above?

Reply:
yes, it seems like 3.6 checks until a certain decimal point vs 3.7, 3.8

Reply:
trying to fix that by only checking until a certain decimal point if a value is float

Reply:
ah, makes sense!